index.html

```diff
@@ -1412,6 +1412,119 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID menthod, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      A reliable guide for making assessments of various technologies and weighing
+      ethical considerations is the 
+      <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>
+      document.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        As with all things, strong consideration for the appropriate and ethical use
+        of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p>
+        When implementing or utilizing a DID method, consideration should be given
+        to the environmental impacts of any underlying technologies.  For example
```

Comment:
I support the guidance that one should give consideration to the environmental impacts of any system you build. However, that example is inappropriate as it does not meet any standard of objective review. Rather, it is a political an ideological attack on PoW systems, an attack which is inappropriate for the W3C and this working group to make.

In the same manner, we *could* demand that people consider the ethical implications of Proof of Stake algorithms which clearly reinforce existing power dynamics leading to a runaway cascade in which the larger, vested players establish an unassailable and therefore, undemocratic authority.

IMO, neither of these are appropriate positions for the W3C and the DID WG to take, precisely because it is not a technical recommendation but a political one.

If some part of the W3C were to develop standards for evaluating the environment impact of technologies, and were willing to apply that standard across the board to all existing and propose technologies, I would welcome that objective, fair framework for discussion.

This is not that.

Reply:
> In the same manner, we _could_ demand that people consider the ethical implications of Proof of Stake algorithms which clearly reinforce existing power dynamics leading to a runaway cascade in which the larger, vested players establish an unassailable and therefore, undemocratic authority.

Good call out.  Will make some adjustments

Reply:
Absolutely agree, and on Tuesday I plan on generating a PR/section for addition that explains why highly secure, deterministic consensus is necessary for delivering the highest level of security to protect digital human rights. This is a hill I'm ready, willing, and well prepared to die on, because the defense of digital human rights is worth far more than 1/4 of the electricity we spend on clothes dryers.

Reply:
@csuwildcat that would be great.  I have adjusted the PR to avoid calling out particular approaches (as obviously these can change over time).  Might be optimal if we call out that similar to making assessments on energy usage and comparing those against other benefits of an approach, we should absolutely make assessments related to potential privacy and security over time and that human rights concerns should outweigh others.